build.gradle

```diff
@@ -33,6 +35,17 @@ googleJavaFormat {
   exclude 'build/**'
 }
 
+if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword') &&
```

Comment:
Ok as @garrettjonesgoogle points out we may need to keep this as-is since this allows someone can clone gax-java & call ./gradlew build without having ossrhUsername/ossrhPassword set.